File: src/com/boarbeard/SpaceAlertApplication.java
package com.boarbeard;

import android.app.Application;

import com.boarbeard.io.ExternalMedia;

public class SpaceAlertApplication extends Application {
	private static SpaceAlertApplication singleton;

	public static SpaceAlertApplication getInstance() {
		return singleton;
	}

	@Override
	public void onCreate() {
		super.onCreate();
		singleton = this;

		ExternalMedia.init(this);
	}
}

File: app/src/main/java/com/boarbeard/SpaceAlertApplication.kt
package com.boarbeard

import android.app.Application
import com.boarbeard.io.ExternalMedia
import timber.log.Timber

class SpaceAlertApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        if (BuildConfig.DEBUG) {
            Timber.plant(Timber.DebugTree())
        }
        ExternalMedia.init(this)
    }
}